src/aiortc/rtcrtpsender.py

```diff
@@ -288,21 +327,80 @@ def _send_keyframe(self) -> None:
         Request the next frame to be a keyframe.
         """
         self.__force_keyframe = True
-
+    
+    def ml_predict(self, frame, counter):
```

Comment:
The issue was related to frame_stamping that happens in `media.py` which appends something to the original frame